examples/asr/librispeech_conformer_rnnt/data_module.py

```diff
@@ -0,0 +1,194 @@
+import os
+import random
+
+import torch
+import torchaudio
+from pytorch_lightning import LightningDataModule, seed_everything
+
+
+seed_everything(1)
```

Comment:
Is this the right place to seed? I would imagine this happen once (and only once) at the very beginning of the CLI entry point.